<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>work</class>
 <widget class="QWidget" name="work">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="workOut">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>128</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Serial port data</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdN_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdTimer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="userDataCollect">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>437</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="userTimersViewer">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>187</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777209</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Timers</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>310</width>
     <height>220</height>
    </rect>
   </property>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="userDataClear">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>437</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Clear</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="usertableWidget">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>300</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="userComunications">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>5</y>
      <width>147</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Communication with Serial Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>64</y>
      <width>115</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed of data collecting</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>60</y>
      <width>83</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>1 ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 ms</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Monitor On</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monitor Off</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="userSendToSerial">
       <property name="text">
        <string>Send to Serial</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>30</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RealTimeMeasure</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>179</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>424</y>
     <width>81</width>
     <height>36</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="userDataReady">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Ready</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="userPortStatus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>4</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>   Port Status</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonMasterOn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>MasterOn</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
